packages/admin/src/styles/globals.scss

```diff
@@ -86,9 +86,15 @@ $govuk-assets-path: $sub_path + '/assets/';
 
 //CSS to override TinyMCE toolbar icons when selected
 .tox-tbtn--enabled {
-  background-color:govuk-colour("blue") !important;
-  color: govuk-colour("white") !important;
+  background-color: govuk-colour('blue') !important;
+  color: govuk-colour('white') !important;
 }
 .tox-tbtn--enabled svg {
-  fill: govuk-colour("white") !important;
-}
\ No newline at end of file
+  fill: govuk-colour('white') !important;
+}
+
+@media (min-width: 40.0625em) {
+  .float-left-sm {
+    float: left;
+  }
+}
```

Comment:
Want to avoid custom CSS like the plague ideally - especially stuff applied everywhere. Can use next.js css modules if you need to apply this to a single page

Reply:
I noticed earlier today there's actaully a prop which should fix this without any additional css.

Reply:
Moved that to a module